app/models/bishop.rb

```diff
@@ -1,5 +1,20 @@
 class Bishop < Piece
 
+	def no_move?(x,y)
+		(x == x_cord) && (y == y_cord) ? true : false
+	end
+
+	def off_board?(x,y)
+		(x > 8 || y > 8 || x < 1 || y < 1) ? true : false
+	end
+
+	def bishop_move (x,y)
```

Comment:
let's name this valid_move?(new_x, new_y) for consistency with others.  I'm sure we will have to edit others for consistency later, too, but I think it would be good if we changed this for now.